content/v2.0/cloud/account-management/offboarding.md

```diff
@@ -48,6 +52,9 @@ For details, see [Export a dashboard](/v2.0/visualize-data/dashboards/export-das
 
 To request a backup of data in your {{< cloud-name "short" >}} instance, contact [InfluxData Support](mailto:support@influxdata.com).
 
-### 3. Contact Support
+### Cancel service
 
-To cancel your subscription, reach out to [InfluxData Support](mailto:support@influxdata.com). Your payment method is charged your final balance immediately upon cancelation of service.
+1. Hover over the Usage icon in the left navigation bar and select Billing.
```

Comment:
Add a nav-icon shortcode here for some visual reference:

~~~
1. Hover over the Usage icon in the left navigation bar and select Billing.

    {{< nav-icon "usage" >}}
~~~